webapp/graphql/Search.js

```diff
@@ -1,12 +1,12 @@
 import gql from 'graphql-tag'
 import { userFragment, postFragment } from './Fragments'
 
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-06-02T13:31:27Z" title="Tuesday, June 2nd 2020, 3:31:27 pm +02:00">Jun 2, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,15 +1,17 @@
 import gql from 'graphql-tag'
-import { userFragment, postFragment } from './Fragments'
+import { userFragment, postFragment, tagsCategoriesAndPinnedFragment } from './Fragments'
```


Why do we need the tagsCategoriesAndPinnedFragment here?
It leeds the cypress tests to fail and I cannot see any reason why we would need this information for the search results.